documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -175,30 +175,32 @@ The timeout for building new Kafka Connect images with additional connectots, in
 This value should be increased when using Strimzi to build container images containing many connectors or using a slow container registry.
 
 `STRIMZI_FEATURE_GATES`:: Optional.
-Disables or enables feature gates to activate new features or change the operator behavior.
-For more details about the feature gates, see xref:ref-operator-cluster-feature-gates-{context}[].
+Enables or disables feature gates that activate new features or change the behavior of the operator.
+For more information about each feature gate, see xref:ref-operator-cluster-feature-gates-{context}[].
 
 [id='ref-operator-cluster-feature-gates-{context}']
 == Feature gates
 
-Strimzi operators support feature gates to turn functionality on and off.
-When applied, feature gates alter the behavior of the operator.
-You can change the default state for each feature gate to enable or disable them.
+Strimzi operators support _feature gates_ to enable or disable certain features and functionality. 
+When enabled, a feature gate alters the behavior of the relevant operator. 
 
-Feature gates have 3 stages of maturity:
+Feature gates have a default state of either _enabled_ or _disabled_. 
+To modify a feature gate's default state, use the `STRIMZI_FEATURE_GATES` environment variable in operator's configuration.  
+You can modify multiple feature gates in one environment variable.
+
+Feature gates have three stages of maturity:
 
 * Alpha — typically disabled by default
 * Beta — typically enabled by default
-* General Availability (GA) — typically enabled by default or always enabled
+* General Availability (GA) — typically enabled by default
 
-Alpha stage features might be unstable, subject to change, experimental, or not sufficiently tested for production use.
-Beta stage features are already well tested and their functionality is not likely to change.
-GA means that a feature is stable and should not change.
-GA features might also become features that are always enabled.
-In this case, the feature gate is removed and it will not be possible to disable the feature.
-Alpha and beta features are removed if they do not prove to be useful.
+Alpha stage features might be experimental or unstable, subject to change, or not sufficiently tested for production use.
+Beta stage features are well tested and their functionality is not likely to change.
+GA stage features are stable and should not change in future. 
+We might remove a GA stage feature gate that has been enabled by default for some time, so disabling the feature will no longer be possible. 
```

Comment:
Are we trying to say the feature is incorporated and is no longer as a feature gate? If so, could be clearer.

Reply:
I'm tying myself in knots with the terminology. I suggest making this a simple note:

NOTE: If a GA feature has been enabled by default for some time, its feature gate might be removed.

Reply:
I'm not sure _If a GA feature has been enabled by default for some time, its feature gate might be removed._ describes well what would happen. In many cases we will decide straight away to remove it when it reaches GA - this is how it is planned for the `ControlPlaneListener` for example.

Reply:
OK. I suggest we keep it very simple:

NOTE: Feature gates might be removed when they reach GA.

Reply:
This means that the feature was incorporated into Strimzi and can no longer be disabled.